sdcard/os/hypriotos.sh

```diff
@@ -52,8 +52,9 @@ cleanup(){
 # Takes an URL (.img.zip file) to download an the name of the downloaded file. Assumes that the extracted and the downloaded file has the same names except for the extension
 generaldownload(){
 
-	# Install unzip if not present
+	# Install unzip and partprobe if not present
 	require unzip unzip
+	require partprobe partprobe
```

Comment:
I'm a bit concerned that `pacman` (on host) doesn't have parted, so this will fail then. But I'll update the `require` script before LGTM'ing this so it won't be a problem.